README.md

```diff
@@ -9,68 +9,51 @@ Development for a bank/accounting interface for the Flux resource manager. Write
 
 ### Build Requirements
 
-flux-accounting requires the following packages to build:
+flux-accounting requires the following Python packages to build:
 
-| centos8       | ubuntu      | version |
-| ------        | --------    | ------- |
-| python3-devel | python3-dev | >= 3.6  |
-| python3-pip   | python3-pip | 20.0.2  |
-| tox           | tox         | 3.15.0  |
+| Package Name | Version |
+| ------------ | ------- |
+| pandas       | 0.25.3  |
```

Comment:
It turns out that python/3.7.2 on LC systems only have pandas version 0.24.1. So I think we should probably loosen this up a bit to >= 0.24.1 (I also need to update the requirements.txt in my PR...whoops).

(Sorry that the other comment wasn't a part of a formal review).